content/blog/2023/web-pwa-native-RWA1.md

```diff
@@ -0,0 +1,136 @@
+---
+{% import 'embed.macros.njk' as embed %}
+
+You have an idea for a custom digital product you’d like to build, and it’s time to decide how to build it. One of your friends swears you won’t succeed unless you build a native mobile app for iPhone. A colleague argues that a responsive web app is the only way to go. Who’s right? Are there alternative options? The decision of how to build your app is quite important – affecting project scope, timeline, and budget – but understanding when to choose one option over another can seem insurmountably difficult.
+
+The great news is that you don’t need to spend years becoming a web developer yourself in order to make a wise choice. In the following three-part series, we’ll dig into the three main options for building mobile apps: Responsive Web Apps (RWAs), Progressive Web Apps (PWAs), and Native Mobile Apps, and how to determine the best option for your project. As you read, think about who your app will serve, what their most pressing needs are, and how your app will meet those needs.
+
+## What are RWAs, PWAs, and Native Mobile Apps?
+Here are the biggest differences in a nutshell.
+
+- **Responsive Web App (RWA)** - An application that runs in a browser, such as Chrome, Firefox, or Safari, and responds to fit any screen size on any device.
+- **Progressive Web App (PWA)** - A responsive web app with additional capabilities that mimic a native mobile app.
+- **Native Mobile App** - An app built specifically for one particular mobile operating system, like iOS or Android.
+
+
+{{ embed.img(
+  src='blog/2023/web-pwa-native-comparison-chart.png',
+  alt='A comparison chart showing the differences between
+  responsive web apps, progressive web apps, and native
+  mobile apps'
+) }}
+
+
+## Responsive Web Apps - Best for Wide Reach
+
+> **TL;DR**
+>
+> If your top priority is to reach as wide a range of
+> people as possible, as quickly as possible, while keeping
+> development and maintenance costs down, a responsive web app is
+> likely the best option for building your digital product.
+
+An RWA is essentially a website that resizes and rearranges its content – responding to fit whatever screen size a person chooses to use. RWAs are built using standard web technologies like HTML, CSS, and JavaScript. They are accessible from any device with a web browser such as Chrome, Firefox, Microsoft Edge, Safari, or a mobile browser.
+
+<div class="contain">
+
+{{ embed.img(
+  src='blog/2023/yes.png',
+  alt='The word yes with a smiley and sparkles',
+  attrs={
+    'class': 'align-left'
+  }
+) }}
+
+## Choose an RWA when…
+### Lower Cost
+*…when it’s important to keep initial development costs low.*
+
+Responsive web apps use a single “codebase.” The term “codebase” is developer speak for the collection of code used to build an application. Compared to native mobile apps, RWAs are relatively inexpensive to develop, requiring just one codebase to be available on any device with a web browser.
```

Comment:
@SondraE do you think we could use quotes around the second codebase only? Because we don't use them on the third one... or maybe only use italics around the one that you are defining?